### ISSUE TITLE
Service Container Pipeline

### ISSUE THREAD

**Issue by AndrewBabbitt97** [recro/medulla#139]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your feature request related to a problem? Please describe the problem.

We need containers to be built and pushed to GHCR.

### Describe the solution you'd like

Container images should be built and pushed yo GHCR as a part of a GitHub action run against the main branch.

### Additional context

N/A

**Comment by keithlogan94** [comment]
Yesterday:
Analysis on medulla, learning the project.
Today, digging into github packages, ghcr.
Blockers, none

**Comment by keithlogan94** [comment]
#### Status for Darius
<b>What did you work on yesterday?</b>
`Analysis on medulla, learning the project.`
<b>What did will you work on today?</b>
`Today, digging into github packages, ghcr.`  
<b>Are there any blockers in your way?</b>
`none`
<b>Are we on track to complete the work by end of the sprint with the sprint goal?</b>
`Didn't ask.`
<b>Has a pull request been created and remote branches up to date?</b>
`NA`
*Pull requests need to be created early, for early review, and to make sure work matches vision for ticket.*

**Comment by dariuswatson** [comment]
## example of use of GITHUB_TOKEN within pipeline to push to ghcr.io:

```bash
name: CI 
on:
  push:
    branches:
      - 'main'
jobs:
  build:
    runs-on: ubuntu-latest
    steps:
      -
        name: Checkout
        uses: actions/checkout@v2
      -
        name: Login to Docker Hub
        uses: docker/login-action@v1
        with:
          registry: ghcr.io
          username: ${{ github.actor }}
          password: ${{ secrets.GITHUB_TOKEN }}
      -
        name: Build and push
        uses: docker/build-push-action@v2
        with:
          context: .
          file: ./Dockerfile
          push: true
          tags: ghcr.io/medulla/container_image:latest
      - 
        name: docker pull
        run: |
          docker pull ghcr.io/medulla/container_image:latest
      - 
        name: docker inspect
        run: |
          docker inspect ghcr.io/medulla/container_image:latest
```

**Comment by keithlogan94** [comment]
#### Status for Darius
<b>What did you work on yesterday?</b>
`got github token working, continued research. push to dockerhub, able to use github token, posted CICD Github actions`
<b>What did will you work on today?</b>
`No pressing tasks on this issue, finish up analysis on any other option, but this is the best recommended. Github actor is whatever triggers CICD. Analysis is about complete`  
<b>Are there any blockers in your way?</b>
`None`
<b>Are we on track to complete the work by end of the sprint with the sprint goal?</b>
`Yes`

**Comment by AndrewBabbitt97** [comment]
Closed in #251
